packages/prisma/src/prisma/schema.prisma

```diff
@@ -140,23 +140,23 @@ model Tag {
 }
 
 model QuestionInstance {
-  id String @id @default(uuid()) @db.Uuid
+  id Int @id @default(autoincrement())
 
   questionData Json
   results      Json
   responses    QuestionResponse[]
 
-  sessionBlock      SessionBlock?    @relation(fields: [sessionBlockId], references: [id])
+  sessionBlock      SessionBlock?    @relation(fields: [sessionBlockId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   sessionBlockId    Int?
-  learningElement   LearningElement? @relation(fields: [learningElementId], references: [id])
+  learningElement   LearningElement? @relation(fields: [learningElementId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   learningElementId String?          @db.Uuid
-  microSession      MicroSession?    @relation(fields: [microSessionId], references: [id])
+  microSession      MicroSession?    @relation(fields: [microSessionId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   microSessionId    String?          @db.Uuid
 
-  question   Question @relation(fields: [questionId], references: [id])
-  questionId String   @db.Uuid
-  owner      User     @relation(fields: [ownerId], references: [id])
-  ownerId    String   @db.Uuid
+  question   Question? @relation(fields: [questionId], references: [id], onDelete: SetNull, onUpdate: Cascade)
```

Comment:
Special case: if a question is deleted, we do not want to delete all of the question instances (we want to allow for this use case, hence the copying of questionData into the instance). \`SetNull\` will ensure that the question property on instances of a deleted question are set to \`null\`.